app/models/audit_activity/accident_or_incident/accident_or_incident_updated.rb

```diff
@@ -0,0 +1,96 @@
+class AuditActivity::AccidentOrIncident::AccidentOrIncidentUpdated < AuditActivity::Base
+  def self.from(*)
+    raise "Deprecated - use UpdateAccidentOrIncident.call instead"
+  end
+
+  def self.build_metadata(accident_or_incident)
+    updates = accident_or_incident.previous_changes.slice(
+      :date,
+      :is_date_known,
+      :product_id,
+      :severity,
+      :severity_other,
+      :usage,
+      :additional_info,
+    )
+
+    {
+      accident_or_incident_id: accident_or_incident.id,
+      updates: updates,
+      type: accident_or_incident.type
+    }
+  end
+
+  def title(*)
+    metadata["type"]
+  end
+
+  def subtitle_slug
+    "Updated"
+  end
+
+  def date_changed?
+    new_date_information?
+  end
+
+  def new_date_information?
+    updates["date"]&.second || updates["is_date_known"].count == 2
```

Comment:
Please use `Hash#dig`, for consistency with the rest of the code:
```suggestion
    updates.dig("date", 1) || updates.dig("is_date_known").count == 2
```

and same for the rest of the file.

Reply:
`&.second` is actually used throughout the audit activity updated classes so I think this is actually consistent. Examples in risk_level_updated.rb, test_result_updated.rb, risk_assessment_updated.rb.